src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -16,17 +22,17 @@ public static void main(String[] args) {
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
 
         Country usa = new Country("USA");
-        CountryService countryService = null; // TODO: initialize this instance
+        CountryService countryService = new CountryServiceImpl(new CountryDaoImpl(sessionFactory));
```

Comment:
You are creating a new instance of CountryService every time you need to use it. This is not efficient. Consider using Dependency Injection or Singleton pattern to avoid this problem.